src/components/POLine/Eresources/MaterialTypeField.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Select } from '@folio/stripes/components';
+
+import normalize from '../../Utils/normalize';
+import { requiredMaterialType } from '../../Utils/Validate';
+
+const MaterialTypeField = ({ name, materialTypes }) => (
+  <FormattedMessage id="ui-orders.dropdown.select">
+    {(placeholder) => (
```

Comment:
is it required explicitly? I suggest to switch on empty place holder as we do have in many places already
https://github.com/folio-org/ui-orders/blob/3fe2d50c0e5cf12ab44acfb2af47bdb4d9b27469/src/components/CheckIn/FieldPieceFormat.js#L21
https://github.com/folio-org/ui-orders/blob/370756a5628cb939b35fe77b62abe853febdf35e/src/components/POLine/FundDistribution/FundDistributionForm.js#L51
https://github.com/folio-org/ui-orders/blob/69888f7e34b6ef7538c82abd8f1d196b475b3352/src/components/POLine/Location/LocationForm.js#L66
https://github.com/folio-org/ui-orders/blob/65b2c622da8f9643f1522ce22e2b2a3556782298/src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js#L62
https://github.com/folio-org/ui-orders/blob/3fe2d50c0e5cf12ab44acfb2af47bdb4d9b27469/src/components/CheckIn/AddPieceModal.js#L97

Reply:
in your case since you have empty option, I don't think you should use placeholder prop at all